### PR TITLE
common/x*: add more information link

### PR DESCRIPTION
**Pull request by syleung**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Relates to issue #6062

Extra Note:
* In the description of related issue https://github.com/tldr-pages/tldr/issues/6062#issue-905809548
  * `linux/xar` is listed, but this file is actually in `common/xar`

### CODE REVIEW

File: pages/common/xkill.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Kill a window interactively in a graphical session.
 > See also `kill` and `killall`.
+> More information: <https://man.openbsd.org/xkill>.
```
Comment (marchersimon) [inline pages/common/xkill.md:5]: Why did you use an `openbsd` man page here. `xkill` wasn't developed by `openbsd` right?
Same with `Xeypher` and `xargs`

Comment (syleung) [reply pages/common/xkill.md]: Sorry about this, learnt something new today.

I've now updated the links for:
* xkill - https://www.x.org/releases/current/doc/man/man1/xkill.1.xhtml
* xepher - https://cgit.freedesktop.org/xorg/xserver/plain/hw/kdrive/ephyr/README
* xargs - https://pubs.opengroup.org/onlinepubs/9699919799/utilities/xargs.html

Comment (marchersimon) [reply pages/common/xkill.md]: `xkill` and `xargs` look good, but I think we should use <https://manned.org/xserver-xephyr> for `xephyr`

Comment (syleung) [reply pages/common/xkill.md]: Thanks @marchersimon for the suggestion, have updated `xephyr` link.

**Review by marchersimon — APPROVED** [review]
Thanks @syleung

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**